post_mortens/Temporada_1.md

```diff
@@ -29,6 +29,18 @@ Essa também é uma questão delicada. Nesse caso, sinto que poderiamos nos insp
 
 Adicionar um campo referente ao pronome pelo qual a pessoa prefere ser chamada.
 
+- **Campo de etnia**
+
+Como queremos ajudar todas as minorias, um outro ponto importante é a etnia. O Brasil é um país muito diverso, com muitas pessoas de outros países e culturas. Além disso, um grupo muito subrepresentado e que sofre muito preconceito são os povos indígenas. Acho importante deixarmos claro que queremos acolher essas pessoas e que teremos vagas específicas para elas.
+
+- **Campo de identidade racial**
+
+Outro pecado que sofremos foi divulgar vagas específicas para pessoas que sofrem racismo, mas não nos atentamos a isso no formulário. Precisou que a Bruna Pinos ficasse olhando as fotos no telegram, o que causou muito desconforto e sentimento de **se eu não fizer ninguém vai se atentar a isso**. Por isso, esse é outro ponto muito importante.
+
+- **Campo de deficiência**
+
+Outro ponto importante é pessoas deficientes, que também sofrem muito preconceitos e são deixadas de lado na área. Vale discutirmos sobre, mas acho que apenas um campo sim/não basta, para que não deixe alguém desconfortável. Podemos deixar o campo em que a pessoa especifique como opcional.
```

Comment:
Sobre essa parte, é importante dizer que o termo certo é Pessoa com Deficiência (PcD, para abreviatura), e um ponto relevante ao perguntar sobre é perguntar a respeito de medidas podem ser tornadas para tornar o conteúdo mais acessível à pessoa, pois existem vários tipos de deficiência que requerem diferentes métodos
Na verdade, essa parte de medidas de acessibilidade pode ser interessades para qualquer participante, pois há pessoas com TDAH, por exemplo, que podem não se autodenominar como PcD, mas que podem sugerir formas de acessibilidade

Reply:
Boa, vou atualizar o documento com o termo correto, @emmenezes !

Acho que sobre essas perguntas, achei ótimo também, é algo que realmente não estamos nos atentando e podemos estar cometendo erros graves relacionados a isso. Porém, fico me perguntando se o formulário de inscrição seria o melhor local para isso, Em teoria, a pessoa ainda não teria sido exposta ao conteúdo da BOSS( a não ser que tivesse estudado sozinha), então talvez a pessoa ainda não tenha criticas e nem sugestões a fazer. O que acha de adicionarmos isso ao formulário da semana? Ai poderiamos colher esse feedback.

Se tiver alguma sugestão sobre como podemos fazer isso, só mandar! :heart:

Reply:
Essa parte de sugestão é, por exemplo:
- Se a pessoa for cega, talvez prefira acessar o código antes da aula para que possa ter conhecimento do que está sendo mostrado na hora da aula
- Se a pessoa for autista, talvez peça que os exemplos sejam mais objetivos e tenham menos metáforas para facilitar a compreensão
Penso em coisas que a pessoa já sabe sobre si e que possam nos ajudar a ter uma experiência mais proveitosa.

E sim! Ter um feedback no formulário da semana vai ser muito importante!

Reply:
Aaaaah, entendi! Gostei, realmente. Vale então termos os dois, esse campo pra pessoa poder já falar o que sabe e outro no formulário pra colhermos feedback para melhorias.

Top, mais tarde vou descrever essa parte no post-mortem! Obrigada!